### PR TITLE
Template literals slides

### PR DESCRIPTION
**Pull request by nachosoloaga**

### CODE REVIEW

File: src/slides/class3/slides/TemplateLiterals.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+              ageStr = 'centenarian';
+            } else {
+              ageStr = 'youngster';
+            }
+          
+            // We can even return a string built using a template literal
+            return \`\${str0}\${personExp}\${str1}\${ageStr}\${str2}\`;
+          }
+          
+          let output = myTag\`That \${ person } is a \${ age }.\`;
```
Comment (JulianPasquale) [inline src/slides/class3/slides/TemplateLiterals.jsx:103]: You don't need parentheses here?

**Review by JulianPasquale — APPROVED** [review]
🎉 

FYI I suggest to point this PR to main branch for the same reason explained in the arrow-functions one

File: src/slides/class3/slides/TemplateLiterals.jsx (hunk 1 above)
Comment (nachosoloaga) [reply src/slides/class3/slides/TemplateLiterals.jsx:103]: Nope, when calling a function with a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates) no parentheses are used.

It's what styled-components (for example) uses:

> Utilising tagged template literals (a recent addition to JavaScript) and the power of CSS, styled-components allows you to write actual CSS code to style your components.

```
const Button = styled.button`
  color: grey;
`;
```

Comment (JulianPasquale) [reply src/slides/class3/slides/TemplateLiterals.jsx:103]: oh gotcha 👏

**Pull request merged**